pkg/monitortestlibrary/allowedalerts/basic_alert.go

```diff
@@ -156,11 +181,11 @@ func (a *alertBuilder) toTests() []AlertTest {
 func (a *basicAlertTest) InvariantTestName() string {
 	switch {
 	case len(a.namespace) == 0:
-		return fmt.Sprintf("[bz-%v][invariant] alert/%s should not be at or above %s", a.bugzillaComponent, a.alertName, a.alertState)
+		return fmt.Sprintf("[%v][invariant] alert/%s should not be at or above %s", a.bugzillaComponent, a.alertName, a.alertState)
```

Comment:
We don't have to deal with it now but this refactor made me think of this.  We should ditch `bz` eventually since we don't use bugzilla.  If one annotates these with [Jira:Etcd] instead for example, ci-test-mapping and component readiness will put new alert tests in the right components. But you'd have to get a PR to ci-test-mapping too to make sure it gets linked up with the old tests, so probably better done separately